src/cluster_legacy.c

```diff
@@ -6552,3 +6552,30 @@ void clusterReplicateOpenSlots(void) {
 
     zfree(argv);
 }
+
+del_stat delUnOwnedKeys(void) {
```

Comment:
This should ideally be in cluster.c. The difference is that cluster_legacy is for low level interface, whereas I think we can just use the high level APIs (see cluster.h).

Reply:
We need to call delKeysInSlot, which is a static/local function defined in this file. Shall we change that then call it from cluster.c?